uint32 num_frames
float64 timeout
---
string success
stalk_detect/stalk stalk
float64 confidence
uint16 num_frames